Frontend/app/src/main/java/io/happybugs/happybugs/activity/ReportActivity.java

```diff
@@ -197,7 +197,7 @@ protected void sendReportData() {
         userReport.put("data", dataArray);
 
         Retrofit rfInstance;
-        rfInstance = RetrofitInstance.getInstance();
+        rfInstance = RetrofitInstance.getInstance(curContext);
```

Comment:
If this is a variable containing current context, maybe just name it currentContext? Or currContext if you want to make it shorter :) (I've seen a lot of examples where current is abbreviated to curr)